ow_plexil/src/plexil-adapter/OwInterface.cpp

```diff
@@ -401,6 +385,12 @@ void OwInterface::managePanTilt (const string& opname,
 }
 
 ///////////////////////// Antenna/Camera Support ///////////////////////////////
+
+bool OwInterface::anglesEquivalent (double deg1, double deg2, double tolerance)
+{
+  return normalize_degrees(fabs(deg1 - deg2)) <= tolerance;
```

Comment:
Now that normalize_degrees is fixed, this should be changed to the form that takes absolute valued after normalizing.

Reply:
Okay, done, and the tests still pass.